compass/ocean/__init__.py

```diff
@@ -43,6 +45,7 @@ def __init__(self):
         self.add_test_group(IceShelf2d(mpas_core=self))
         self.add_test_group(IsomipPlus(mpas_core=self))
         self.add_test_group(MerryGoRound(mpas_core=self))
+        self.add_test_group(MitgcmBaroclinicGyre(mpas_core=self))
```

Comment:
I'd be inclined to call this test group `BaroclinicGyre` rather than `MitgcmBaroclinicGyre`. I think shorter is better, and I doubt we'll be implementing multiple baroclinic gyre cases.

Reply:
I'm fine with the shorter name.  @alicebarthel, do you remember why we settled on the longer name?  Presumably because we did think we might have other baroclinic gyre tests?

Reply:
I like the idea of a more general (and shorter) name, unless it significantly increases the scope of this new test case. 
As I remember it: as a dev step, I preferred to err on the side of too specific because I was not sure how much of the set-up was going to be hard-coded vs. generalized, or whether other baroclinic cases would be developped in the future. We hoped to have a narrowly defined scope of reproducing the MITgcm test case - so went with the specific name. 
As it currently stands, I could see this becoming a general baroclinic gyre case if we include options to set T,S in the config file. Does it matter that it is on spherical coordinates (vs. barotropic on planar)? Is there anything else that seems specific enough to either include in the name or at least in the description?

Reply:
I think that you can consider it a general baroclinic gyre test case and go with the shorter name. The way I see it, it is the responsibility of whomever might want to develop a baroclinic gyre with different properties to change the name at that point. The planar vs. spherical distinction can also happen down the line (e.g., as it is done for the `global_convergence` test case where the path of the generated test case includes `planar` vs. `spherical`). 

I think it will be clear that the default configuration is the one that is meant for comparison with the MITgcm output (if you find a way to include that output in the analysis). Or maybe it will only be apparent in the documentation, where you have references and maybe copy an image from the MITgcm page documenting their results. Or maybe you change the `default` test name to `mitgcm_comparison`. I'm not sure what the best route is at this point. @alicebarthel Maybe you have preferences here?

Reply:
Renamed the case. I like the idea of changing the default test name to mitgcm_comparison, so something to keep in mind for later.